docs/getting-started.md

```diff
@@ -207,22 +216,23 @@ First, a few notes about versioning - Bit doesn’t use semantic versioning. Ins
 
 ### Let's say we want to add documentation to the component we've just created.
 
-1\. Import the component to the inline_bits folder:
+1. Import the component to the inline_bits folder:
```

Comment:
> Import the component to the inline_bits folder:

Import the component to the **inline_components** **directory:**

We use the folder/directory terms interchangeably, and I rather use `directory`, `folder` feels more GUI oriented.